provider/resource_config_system.go

```diff
@@ -78,12 +83,17 @@ func resourceConfigSystemRead(d *schema.ResourceData, m interface{}) error {
 	if err != nil {
 		return fmt.Errorf("Error getting system configuration %s", err)
 	}
+	storage := jsonData.StoragePerProject.Value
```

Comment:
We should accept the user input for storage quota in GB and then only convert it to bytes while making the API call?

Reply:
As far as I’m aware the api accept bytes. Did the converter to GB since we use it in other places as well which is an argument to DRY this part as well.